3.2/Dockerfile

```diff
@@ -67,12 +67,15 @@ RUN buildDeps=' \
 		libsqlite3-dev \
 		make \
 		patch \
+		libgmp-dev \
+		build-essential \
+		libc6-dev \
 	' \
 	&& set -ex \
 	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/* \
 	&& bundle install --without development test \
-	&& for adapter in mysql2 postgresql sqlite3; do \
+	&& for adapter in sqlserver mysql2 postgresql sqlite3; do \
```

Comment:
I'd prefer to keep these in the same order they are in the entrypoint, so `sqlserver` should go between `postgresql` and `sqlite3`. :+1:

Reply:
No problem.